### PR TITLE
fix: Renaming google plus entry to google groups in admin content social link page

### LINKED ISSUE #3584: Rename google plus entry to google groups from admin social link page
Remove google plus entry from admin/content/social link page.
Screenshot from 2019-10-30 10-00-11

**Additional context**
If this is necessary , i am working on this.

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #3584

Short description of what this resolves:
It changes the google plus entry to google groups in admin/content page .Also fixes the required regex expression for google goups url (https://groups.google.com/forum/#!forum/open-event).Also updated the warning messages shows in box after filling wrong url from "enter a valid google plus url" to "enter a valid google groups url"

Changes proposed in this pull request:
-changed google plus entry in (https://eventyay.com/admin/content) to google groups entry.

Screenshot from 2019-10-31 18-00-54
on entering the wrong url -
Screenshot from 2019-10-31 18-19-13
on entering the right url -
Screenshot from 2019-10-31 18-19-53

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by kushthedude** [comment]
@maze-runnar It looks good, I have pushed a text commit to `Successfully Deploy` netlify.

**Comment by kushthedude** [comment]
@maze-runnar Can you also rename the `Google Plus` to `Google Groups` in the complete models and input id?
You will also need to change the icon for the same. In the meantime you can convert this PR to `WIP`

**Comment by maze-runnar** [comment]
> @maze-runnar It looks good, I have pushed a test commit to `Successfully Deploy` netlify.

Thanks @kushthedude

**Comment by maze-runnar** [comment]
> @maze-runnar Can you also rename the `Google Plus` to `Google Groups` in the complete models and input id?
> You will also need to change the icon for the same. In the meantime you can convert this PR to `WIP`

I have changed input_id  , But i am not sure about models  , can you please guide me .

**Comment by kushthedude** [comment]
> > @maze-runnar Can you also rename the `Google Plus` to `Google Groups` in the complete models and input id?
> > You will also need to change the icon for the same. In the meantime you can convert this PR to `WIP`
> 
> I have changed input_id , But i am not sure about models , can you please guide me .

In `app/models`

File: app/templates/components/forms/admin/content/social-links-form.hbs [hunk 1]
```diff
@@ -21,7 +21,7 @@
     }}
   </div>
   <div class="field">
-    <label>{{t 'Google +'}}</label>
+    <label>{{t 'Google Groups'}}</label>
     {{widgets/forms/link-input
       segmentedLink=socials.segmentedGoogleUrl
       inputId='google_plus'
```
Comment (kushthedude) [inline app/templates/components/forms/admin/content/social-links-form.hbs]: Rename here too

**Comment by maze-runnar** [comment]
> > > @maze-runnar Can you also rename the `Google Plus` to `Google Groups` in the complete models and input id?
> > > You will also need to change the icon for the same. In the meantime you can convert this PR to `WIP`
> > 
> > 
> > I have changed input_id , But i am not sure about models , can you please guide me .
> 
> In `app/models`

ok , thanks

File: app/templates/components/forms/admin/content/social-links-form.hbs (hunk 1 above)
Comment (maze-runnar) [reply app/templates/components/forms/admin/content/social-links-form.hbs]: ok

**Comment by maze-runnar** [comment]
@kushthedude  sir i have made changes. Do i need to make any further changes ?

File: app/models/user.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ export default ModelBase.extend({
   facebookUrl   : attr('string'),
   instagramUrl  : attr('string'),
   twitterUrl    : attr('string'),
-  googlePlusUrl : attr('string'),
+  googleGroupsUrl : attr('string'),
```
Comment (kushthedude) [inline app/models/user.js]: Dont change this

Comment (maze-runnar) [reply app/models/user.js]: ok sir .

File: app/templates/components/forms/admin/content/social-links-form.hbs [hunk 3]
```diff
@@ -21,10 +21,10 @@
     }}
   </div>
   <div class="field">
-    <label>{{t 'Google +'}}</label>
+    <label>{{t 'Google Groups'}}</label>
```
Comment (maze-runnar) [reply app/templates/components/forms/admin/content/social-links-form.hbs]: ok sir .

Comment (CosmicCoder96) [inline app/templates/components/forms/admin/content/social-links-form.hbs]: Why Plural?
`Groups` implies you can add multiple groups, but there is only one text field. Keep it `Group`

**Comment by kushthedude** [comment]
@maze-runnar Please change ~groups~ group

File: app/templates/components/forms/admin/content/social-links-form.hbs [hunk 4]
```diff
@@ -21,10 +21,10 @@
     }}
   </div>
   <div class="field">
-    <label>{{t 'Google +'}}</label>
+    <label>{{t 'Google Group'}}</label>
     {{widgets/forms/link-input
       segmentedLink=socials.segmentedGoogleUrl
-      inputId='google_plus'
+      inputId='google_groups'
```
Comment (kushthedude) [inline app/templates/components/forms/admin/content/social-links-form.hbs]: Change here too else validation wont work

**Comment by maze-runnar** [comment]
I have changed google groups to google group but i think its actual name is google groups not group.
As in url it is  **groups.google.com
Screenshot from 2019-11-03 19-10-24
**

**Pull request merged**